data/src/cls/Pet.cls

```diff
@@ -0,0 +1,41 @@
+/// PURPOSE: This class stores information about pets. To populate 100 sample pets, you can run: do ##class(Demo.Pet).Populate(100)
+Class Demo.Pet Extends (%Persistent, %Populate)
+{
+
+Property Name As %String(POPSPEC = "##class(%PopulateUtils).FirstName()");
+
+Property Type As %String(VALUELIST = ",dog,bird,cat,rat,mouse,rabbit");
+
+Property Color As %String;
+
+Property PetShop As Demo.PetShop;
+
+Index ColorIdx On Color [ Type = bitmap ];
```

Comment:
is this the correct way to name indices?  Could these be "ColorIndex"? Or are these changes out of scope?
```suggestion
Index ColorIndex On Color [ Type = bitmap ];
```

Reply:
Thank you @zkrowiak. Your suggested changes make sense to me. I will talk to Jenny about it.